### ISSUE TITLE
University of Antananarivo [via DMNS] Paleo - Institution Acronym

### ISSUE THREAD

**Issue by acdoll** [ArctosDB/new-collections#248]
Institution to register at [GRSciColl](https://www.gbif.org/grscicoll/institution/search)

**Comment by acdoll** [comment]
Requested to use UA as institutional acronym - according to GRSciColl, this is already claimed by [University of Alabama](https://www.gbif.org/grscicoll/institution/1679d024-9c39-4eb2-a0e5-42ad4ac31252). 
The University of Antananarivo is already registered under [at least two](https://www.gbif.org/grscicoll/institution/search?q=Antananarivo&offset=0) separate acrynyms. The name "University of Antananarivo" is tied to the acronym ZUAC which appears to be specifically associated with an Arthropod collection. There is also UADBA with the name "University dAntananarivo, Department de Biologie Animale" which apparently refers to a vertebrate collection.
I'm not sure if we can resolve this confusion between institution and collection or if it's best to just create a new institution to specific to this paleo collection. Thoughts?

**Comment by dustymc** [comment]
The "DWC Triplet" is not and cannot be unique at any useful scope. There are a half-dozen UAM:Mamm (and hundreds of "UAM") out there, that's been used in the literature in reference to multiple things, and there's just no way to un-do that.

What's now GRSciColl was started as an attempt to make communication between things like Arcos and GenBank easier. GenBank quickly abandoned the idea and built their own registry, and I can't understand what GRSciColl is trying to do since. A registry is a fabulous idea, but one that refuses to recognize that some arbitrary string isn't going to be unique at any useful scale can't be useful to us.

http://arctos.database.museum/guid/UA:whatever:whatever is globally unique and I have absolutely no problem with you claiming it for Antananarivo.

**Comment by campmlc** [comment]
Maybe we should register at GenBank.

**Comment by dustymc** [comment]
You absolutely should register at GenBank, but there's no reason for that to control guid_prefix either.

https://github.com/ArctosDB/arctos/issues/2413

http://www.insdc.org/controlled-vocabulary-specimenvoucher-qualifier

**Comment by campmlc** [comment]
From NCBI:
Sounds like if you get your acronym in first you are set. Subsequent
duplicates get country and state codes.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5824777


When more than one institution uses the same code for their specimen, the
International Organization for Standardization three letter country code is
used to unique the collections. If the institutions are from the same
country, a state code is added in addition to country code. The institution
code that is already in the database is retained (without the country code)
and the subsequent ones are registered with country codes (state codes
where applicable).

For example, all the following institutions use UAM as their institution
code. To distinguish between the collections, the institution codes are
listed as:

University of Alaska, Museum of the North UAM

University of Arkansas at Monticello UAM<USA-AR>

University of Alabama, Malacology Collection UAM<USA-AL>

Universidad Autonoma De Madrid culture collection of cyanobacteria UAM<ESP>

Universidad de los Andes, Facultad de Ciencias UAM<VEN>

Since University of Alaska, Museum of the North (UAM) was the first one to
be registered in the BioCollections Database, UAM is retained for
University of Alaska and the subsequent UAM codes are added with country
and state codes. When a record is submitted to Genbank with an ambiguous
code (ex: UAM), it prompts a consult so a curator can confirm the correct
institution is listed.

**Comment by dustymc** [comment]
Yea I'm not a big fan of the values issued by GenBank, but it's still a functional system - one which realizes that "UAM" is used for all sorts of things and so provides a unique way of saying "*that* UAM."

Best case, use something like Arctos' GenBank builder tool for your GB submissions and the code is entirely internal where you never need to see it. Worst case, submit as "UAM" and the various UAM folks should get a 'is this yours?' email. Either way, magic happens at GenBank, the Arctos crawler finds those submissions, yay everybody - the system works despite the wonky format of the identifiers.

**Comment by acdoll** [comment]
Thanks for your comments. They will be very happy to hear they can use UA for their Arctos collections. I will refer them to the NCBI BioCollections Database and get them to register the collection there. I don't think they'll be generating GenBank sequences anytime in the near future but registering here could be one additional step towards formalizing their nascent  museum.

**Comment by acdoll** [comment]
@dustymc I see that two of the new collections use 'Paleo' for the collection code instead of 'ES'. Does that cause an issue with collection specific code tables?

**Comment by dustymc** [comment]
Not sure what you mean - there's no paleo in http://arctos.database.museum/info/ctDocumentation.cfm?table=CTCOLLECTION_CDE

If you mean as part of guid_prefix, that's not a problem (and is separate from collection_cde in Arctos, which probably needs a new name...).

http://handbook.arctosdb.org/documentation/catalog.html#guid-prefix
http://handbook.arctosdb.org/documentation/catalog.html#collection-code

**Comment by campmlc** [comment]
How about collectiontype_cde

**Comment by acdoll** [comment]
Yes, I meant in the GUID_PREFIX. Sorry, I mis-read what was on the new collection setup page and interpreted it as the exact opposite of what was written.
